apps/condo/domains/user/schema/ForgotPasswordService.test.js

```diff
@@ -165,5 +170,92 @@ describe('ForgotPasswordAction Service', () => {
             expect(result.errors).toHaveLength(1)
             expect(result.errors[0].message).toEqual(`${PASSWORD_TOO_SHORT}] Password is too short`)
         })
+        
+        it('can change password with ConfirmPhoneAction', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [user, userAttrs] = await createTestUser(admin)
+            const client = await makeClient()
+
+            const [{ token }] = await createTestConfirmPhoneAction(admin, {
+                phone: userAttrs.phone,
+            })
+            const [actionBefore] = await ConfirmPhoneAction.getAll(admin, { token })
```

Comment:
it's unstable test!

Reply:
You already have this object and you don't need to use getAll. Use object from 179 line